slime/backends/fsdp_utils/actor.py

```diff
@@ -323,34 +365,27 @@ def train(self, rollout_id, rollout_data_ref):  # type: ignore[override]
             if self.args.use_kl_loss:
                 reported["kl_loss"] = kl_loss.detach()
 
-            loss = loss / grad_accum
+            # Scale loss for gradient accumulation
+            loss = loss * dist.get_world_size() / self.args.global_batch_size
             loss.backward()
-
+            clear_memory()
```

Comment:
hmm.. this could make the training really slow could you check if removing the clear_memory here will trigger oom?